src/epfl-post-teaser/index.js

```diff
@@ -2,13 +2,14 @@ import InspectorControlsPostTeaser from './inspector'
 
 const { __ } = wp.i18n
 const { registerBlockType } = wp.blocks
+const { CheckboxControl } = wp.components
 const { Fragment } = wp.element
 
 registerBlockType(
 	'epfl/post-teaser',
 	{
 		title: __( "EPFL Post Teaser", 'epfl'),
-		description: 'v1.0.2',
+		description: 'v1.0.3',
```

Comment:
Plus un changment mineur qu'un bugfix

```suggestion
		description: 'v1.1.0',
```